README.md

```diff
@@ -148,6 +148,7 @@ MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-a
 | workers\_group\_launch\_template\_defaults | Override default values for target groups. See workers_group_defaults_defaults in locals.tf for valid keys. | map | `{}` | no |
 | write\_aws\_auth\_config | Whether to write the aws-auth configmap file. | string | `"true"` | no |
 | write\_kubeconfig | Whether to write a Kubectl config file containing the cluster configuration. Saved to `config_output_path`. | string | `"true"` | no |
+| iam_path | If provided, all IAM roles will be created with path. | string | `""` | no |
```

Comment:
I think this is missing `\` for the underscore. See https://github.com/terraform-aws-modules/terraform-aws-eks#doc-generation

Reply:
This is because the version of `terraform-docs` is not new enough. Update `terraform-docs` and rerun `pre-commit run -a` to get this fixed.

Reply:
I updated the README so it follows the convention of escaped underscores.